core/src/main/java/nl/nn/adapterframework/http/HttpSender.java

```diff
@@ -676,6 +676,7 @@ public void setPostType(String type) throws ConfigurationException {
 	}
 
 	@IbisDoc({"When false and <code>methodeType=POST</code>, request parameters are put in the request body instead of in the url", "true"})
+	@Deprecated
 	public void setParamsInUrl(boolean b) {
```

Comment:
Het zou wel goed/mooi zijn als je op een of andere manier ook dynamische request-parameters aan de url zou kunnen toevoegen.

Reply:
Dit gaat over query params niet path params